packages/cli/generators/project/templates/package.json.ejs

```diff
@@ -94,11 +94,14 @@
     "preopenapi-spec": "npm run build",
     "openapi-spec": "node ./dist/openapi-spec",
 <% if (packageManager === 'yarn') { -%>
-    "prestart": "yarn run build",
+    "rebuild": "yarn run clean && yarn run build",
+    "prestart": "yarn run rebuild",
+    "start": "yarn run prestart && node -r source-map-support/register .",
```

Comment:
should I remove the `yarn run prestart` call from here then @raymondfeng, in order to address enhancing yarn users experience in a separate PR ?

Reply:
+1. I assume `yarn` also works with implicit `prestart`, right?

Reply:
yes for `yarn 1`, but it seems that `yarn 2` does not instead (pls refer to @mdbetancourt [link](https://yarnpkg.com/getting-started/migration#explicitly-call-the-pre-and-post-scripts)). I think we currently have an inconsistent behaviour concerning yarn users, since we are using hooks.

Reply:
If that's the case, we can add `yarn 1` requirement to `engines` - see https://classic.yarnpkg.com/en/docs/package-json/#toc-engines.

Reply:
would it be better to address this in a separate issue as well ?

Reply:
this is also the reason why @mdbetancourt suggested to avoid the use of `pre`, `post` hooks in order to make the behaviour consistent along different project managers